pkg/skaffold/runner/timings.go

```diff
@@ -56,17 +56,26 @@ func (w withTimings) Build(ctx context.Context, out io.Writer, tagger tag.Tagger
 	return bRes, nil
 }
 
-func (w withTimings) Deploy(ctx context.Context, out io.Writer, builds []build.Build) error {
+func (w withTimings) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact) ([]deploy.Artifact, error) {
 	start := time.Now()
 	fmt.Fprintln(out, "Starting deploy...")
 
-	err := w.Deployer.Deploy(ctx, out, builds)
-	if err != nil {
-		return err
+	dRes, err := w.Deployer.Deploy(ctx, out, builds)
+	if err == nil {
```

Comment:
this error goes unchecked

```golang
dRes, err := w.Deployer.Deploy
if err != nil {
  return nil, err
}
fmt.Fprintln
return dRes, nil
```

Reply:
good catch, got the logic slightly wrong here. that's equivalent to 
```golang
dRes, err := w.Deployer.Deploy
if err == nil {
  fmt.Fprintln
}
return dRes, err
```
which is what I was trying to write.